crates/weaver_resolver/README.md

```diff
@@ -33,14 +35,80 @@ involves the following steps:
 - Resolve iteratively all semantic conventions. This involves the maintenance
   of an unresolved semantic convention list and a resolved semantic convention
   list. The resolution process involves the following steps:
-  - Resolve iteratively all attributes `ref` until no more resolvable `ref` are
-    found.
   - Resolve iteratively all `extends` parent/child clauses until no more
     resolvable `extends` are found. The extended entity inherits prefix,
     attributes, and constraints from the parent entity.
+  - Resolve iteratively all attributes `ref` until no more resolvable `ref` are
+    found.
 - Apply constraints `any_of` and `include`.
 - Validate the resolved semantic conventions
   - No more unresolved `ref` or `extends` clauses. The unresolved list should
     be empty.
   - All constraints satisfied.
 
+## Lineage
+
+The resolution process can optionally compute the lineage for each attribute of
+a semantic convention registry. The lineage as such is not part of the syntax
+and structure of a semantic convention; rather, it's an extension produced by
+the `weaver` tool, intended for use in scenarios such as:
+- A semconv author wishes to verify the exact path followed by the resolution
+process in the case of a complex cascade of inheritance across multiple levels
+between groups.
+- A documentation process aims to add lineage information to the documentation
+of each attribute to allow readers to understand where the definitions of each
+field were defined.
+- The lineage information of a semantic convention could eventually feed into
```

Comment:
So this an experimental feature (today).  I was curious if you wanted to include this in application-telemetry-schema.

This may require some discussion as I might want this to be a defacto part of "resolved telemetry schema" so provenance isn't an opt-in.  For now, this is fine, as the whole project is still experimental.

Reply:
Ok I will mark this feature as experimental in the documentation and I will add a note to precise that the lineage should be part of the application telemetry schema.